packages/contracts/src/contracts/current/protocol/Exchange/MixinSignatureValidator.sol

```diff
@@ -33,7 +33,7 @@ contract MixinSignatureValidator is
 {
     // Personal message headers
     string constant ETH_PERSONAL_MESSAGE = "\x19Ethereum Signed Message:\n32";
-    string constant TREZOR_PERSONAL_MESSAGE = "\x19Ethereum Signed Message:\n\x41";
+    string constant TREZOR_PERSONAL_MESSAGE = "\x19Ethereum Signed Message:\n\x20";
```

Comment:
I think this change makes sense. Using `\x20` works with how we sign Trezor messages in `signature_utils`. It also makes sense after reading [this](https://github.com/trezor/trezor-mcu/issues/163) and [this](https://github.com/AdExNetwork/adex-core/search?q=%5Cx19Ethereum+Signed+Message%3A%5Cn%5Cx20&unscoped_q=%5Cx19Ethereum+Signed+Message%3A%5Cn%5Cx20)

Reply:
@hysz were you able to test this against an actual Trezor device?